packages/ui/package.json

```diff
@@ -50,7 +50,7 @@
     "scripts": {
         "start": "react-scripts start",
         "dev": "react-scripts start",
-        "build": "react-scripts build",
+        "build": "env CI=false react-scripts build",
```

Comment:
when I yarn build locally `env env' is not recognized as an internal or external command,
operable program or batch file.`

Reply:
try the latest commit